keyboards/mode/m256ws/info.json

```diff
@@ -0,0 +1,113 @@
+{
+    "manufacturer": "Mode Designs",
+    "keyboard_name": "M256W-S",
+    "maintainer": "Gondolindrim",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "lto": true,
+        "mousekey": false,
```

Comment:
```suggestion
        "mousekey": true,
```

Reply:
Committed in 7d03c51acbfc2df29fa3c63d4286da0e6453417a